src/SourceGenerators/Uno.UI.Tasks/Properties/AssemblyInfo.cs

```diff
@@ -2,3 +2,4 @@
 using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
+[assembly: InternalsVisibleTo("Uno.UI.Tests")]
```

Comment:
It looks like as if this doesn't have any effect. @MartinZikmund Is this something you can help with? Probably this is something I don't clearly see.

Reply:
At first glance I don't see a reason why it shouldn't work. Let's see what CI says on the last change you made

Reply:
It's failling differently:

> src\Uno.UI.Tests\Resources\Given_ResourceCandidate.cs(63,28): Error CS0433: The type 'ResourceCandidate' exists in both 'Uno.UI.Tasks.v0, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null' and 'Uno, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null'

I'll workaround that by adding `<Compile Include` in the unit test project.